common/utilities/integration_adaptors_logger.py

```diff
@@ -90,6 +90,8 @@ def _format_and_write(self, message, values, process_key_num, level):
             message += f' CorrelationId={correlation_id.get()}'
         if inbound_message_id.get():
             message += f' InboundMessageId={inbound_message_id.get()}'
+        if interaction_id.get():
+            message += f' InteractionId={interaction_id.get()}'
```

Comment:
Unittests for logger would need updating to test this.

Reply:
Added test to assert it's attached to the log message correctly